src/navigators/AppStack.js

```diff
@@ -17,17 +22,38 @@ const AppStack = () => (
     <Stack.Screen
       name={MAIN_SCREEN}
       component={MainScreen}
-      options={{ title: strings.MAIN_SCREEN.title }}
+      options={({ navigation }) => ({
+        title: strings.MAIN_SCREEN.title,
+        headerRight: () => (
+          <TouchableOpacity
+            onPress={() => navigation.push(CONVERSATIONS_SCREEN)}
+            style={styles.rightButton}>
+            <Image source={conversationsIcon} />
+          </TouchableOpacity>
```

Comment:
What about moving this to a separate component `<ChatButton/>`? In order to make this file cleaner